### PR TITLE
adds meta tags for google, twitter and OG

### PR DESCRIPTION
**Pull request by housseindjirdeh**
I think the image you created is perfect for sharing, so used that for meta tags (but let me know if you think otherwise!)

### CODE REVIEW

**Comment by cheeaun** [comment]
Looks like new phone frames there 😉

File: site/_includes/head.html [hunk 1]
```diff
@@ -7,6 +7,28 @@
...
+  <meta itemprop="image" content="{{ "/assets/images/hnpwa-showcase.png" | relative_url }}">
+
+  <!-- Twitter Card data -->
+  <meta name="twitter:card" content="summary_large_image">
+  <meta name="twitter:site" content="{{ site.twitter_username }}">
+  <meta name="twitter:title" content="{{ site.title }}">
+  <meta name="twitter:description" content="{{ site.description }}">
+  <meta name="twitter:creator" content="{{ site.twitter_username }}">
+  <!-- Twitter summary card with large image must be at least 280x150px -->
+  <meta name="twitter:image" content="{{ "/assets/images/hnpwa-showcase.png" | relative_url }}">
```
Comment (markgoho) [inline site/_includes/head.html]: It looks like the Twitter card also supports `twitter:image:alt` for accessibility. Could we add that in?

Comment (housseindjirdeh) [inline site/_includes/head.html]: Cheers :raised_hands:. Just added it in!

**Comment by housseindjirdeh** [comment]
@cheeaun :stuck_out_tongue: yeah @addyosmani set that image up. I'll update the site this week hopefully so all the implementations share the same phone frame!

**Comment by addyosmani** [comment]
This looks great. Could we stage a quick deploy of this just so we can sanity check the meta-tags? Changes otherwise are 👍. Thanks for working on this, @housseindjirdeh!

**Comment by housseindjirdeh** [comment]
:man_dancing: https://hnpwa-watermelon-city.surge.sh/

You can see how the twitter card looks [here](https://cards-dev.twitter.com/validator).

**Pull request merged**